.github/workflows/build_appimage.sh

```diff
@@ -484,18 +484,26 @@ EOF
     libxkbcommon.so.0
   )
 
-  # fix AppImage output file name
+  # fix AppImage output file name, maybe not needed anymore since appimagetool lets you set output file name?
   sed -i 's/Name=qBittorrent.*/Name=qBittorrent-Enhanced-Edition/;/SingleMainWindow/d' /tmp/qbee/AppDir/usr/share/applications/*.desktop
 
-  APPIMAGE_EXTRACT_AND_RUN=1 \
-    /tmp/linuxdeployqt-continuous-x86_64.AppImage \
+  export APPIMAGE_EXTRACT_AND_RUN=1
+  /tmp/linuxdeployqt-continuous-x86_64.AppImage \
     /tmp/qbee/AppDir/usr/share/applications/*.desktop \
     -always-overwrite \
-    -appimage \
+    -bundle-non-qt-libs \
     -no-copy-copyright-files \
-    -updateinformation="zsync|https://github.com/${GITHUB_REPOSITORY}/releases/latest/download/qBittorrent-Enhanced-Edition-x86_64.AppImage.zsync" \
     -extra-plugins="$(join_by ',' "${extra_plugins[@]}")" \
     -exclude-libs="$(join_by ',' "${exclude_libs[@]}")"
+
+  # Workaround to use the static runtime with the appimage
+  export ARCH=x86_64
```

Comment:
```sh
ARCH="$(arch)"
```

and `export` can be omitted because this variable only used in current shell, not for sub process.

And we may support aarch64 AppImage in the future. So we may not limit `ARCH=x86_64`

Reply:
Are you sure I shouldnt use the export? I've had this issue with go-appimage where if I didnt export ARCH it would error out saying that it could not detect the architecture.

Now it is likely that this appimagetool doesnt even check that, but I added it just in case, let me know if I should remove the export.

Reply:
Ah. Maybe `goappimage` uses `ARCH` environment variable to check arch, but `appimagetool` may not.

From your PR `ARCH` only used in `appimagetool_download_url="https://github.com/AppImage/appimagetool/releases/download/continuous/appimagetool-${ARCH}.AppImage"`

So I guess you don't need `export`. But it doesn't matter to keep `export`.

Reply:
Thanks @abcfy2 I will remove it and make sure that the build goes thru, the only detail is that right now I am on the phone because I dont have electricity, so hopefully it will be soon